imap_processing/cli.py

```diff
@@ -596,7 +595,7 @@ def do_processing(
                 / "imap_his_pset-calibration-prod-config_20240101_v001.csv"
             )
             hi_dependencies[0] = load_cdf(hi_dependencies[0])
-            datasets = [hi_l1c.hi_l1c(hi_dependencies, self.version)]
+            datasets = hi_l1c.hi_l1c(hi_dependencies, self.version)
```

Comment:
@subagonsouth this was different than your other functions which all returned `list[xr.Dataset]` so mocking it didn't work. I changed this to be the same return type. Do you want this to be a solo Dataset instead, or is this OK? Feel free to push changes or suggest what you'd like.